### PR TITLE
Angular 13 upgrade

### PR DESCRIPTION
**Pull request by apenalver2**
Hi all,

Upgrade the angular(to v13) and other dependencies to the last version.
#56 

image
image

Any comment is welcome

Thank you

### CODE REVIEW

**Comment by apenalver2** [comment]
Can someone help me fix why when Travis is doing the build is crashing?

Thank you

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-preferences",
-  "version": "0.0.0",
+  "version": "10.0.0",
```
Comment (alfonsserra) [inline package.json:3]: Please do not change this one. Change the Library one to be published.

**Review by alfonsserra — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (alfonsserra) [reply package.json:3]: Please change this one: https://github.com/systelab/systelab-preferences/blob/angular-13-upgrade/projects/systelab-preferences/package.json

**Review by alfonsserra — APPROVED** [review]

**Review by oGalceran1 — APPROVED** [review]
Thanks

**Comment by olgapuig** [comment]
This PR is closed without merging. The Angular update from 12 to 14 is being done at https://github.com/systelab/systelab-preferences/issues/56

**Pull request closed without merging**